frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -56,6 +59,43 @@ class PopupCreateFile extends React.Component {
     document.removeEventListener(CUSTOM_EVENT.APP_CUSTOM_EVENT_LISTENER, this.customEventReducer)
   }
 
+  componentDidUpdate (prevProps, prevState, snapshot) {
+    const { state, props } = this
+
+    if (state.uploadedFiles.length === state.uploadFiles.length && state.uploadStarted) {
+      let uploadedFiles = state.uploadedFiles
+      this.setState({ uploadStarted: false })
+
+      uploadedFiles = uploadedFiles.filter(f => f.jsonResult.code !== 200)
```

Comment:
your variable should be named something including "failed" like failedUploadFileList